environment_setup/setup_software.sh

```diff
@@ -80,6 +76,7 @@ if [[ $(lsb_release -rs) == "20.04" ]]; then
     host_software_packages+=(llvm-6.0)
     host_software_packages+=(libclang-6.0-dev)
     host_software_packages+=(libncurses5)
+    host_software_packages+=(qt5-default)
```

Comment:
I thought we were aiming to remove qt5 dependency?

Reply:
I tried to but ErForce uses some deprecated functions that Qt6 doesn't have and I couldn't bother fixing them.

I think the solution is to re-write er-force sim to not use Qt at all and remove the qt dependencies